dev-meetings/2025/2025-09-05/README.md

```diff
@@ -1,8 +1,191 @@
+AS: D you have a list?
+DM: We have prelim. list of science cases but still have to go through it.
+AS: Regarding what gammapy can not do as well?
+DM: We have to go through it and I will give an update.
+AS: Would be very important.
+AS: Is #4644 "Generation of a pdf version of our documentation" supported? DM: Probably yes.
+
+RT joins the meeting.
+
+#6128 "Prototype for a new ObservationTable class"
+------------------
+LS reports and shows attached jupyter-notebook, with which the class can be tested. The prototype introduces a new internal data format for the observation table and a reader, that can recognize the data format on disk and call a corresponding converter function. This way, the internal table is separated from the gadf format. For now, only a gadf->internal format converter is implemented. In all of gammapy, some tests still, should be able to fix. We wanted to ask, if you have any feedback on the design and the implementation.
+
+AS: Does the code work also for AltAz instruments, as these columns are not specified in internal model?
+LS: Yes, they are loaded as optional columns and not modified by the converter, as before.
+AS: Would it be good to also implement into the internal model, analog to RADEC, which units they should have?
+LS: Yes agree, thank you!
+LS: A question I came up with is the reason/relevance of making a unique-table in data_store, which I had to adapt in l.84.
+RT, MF, AS dicuss. 
+AS: A table containing only one entry should also be supported.
+LS: Agree, thank you, I will take care of this.
+
+RT: Question, if in internal model, the OBS_ID is unique, maybe we need separate, unique identifier unequal to the OBS_ID.
+MF: For CTAO it is even a problem, in our definition OBS_ID would connect to time of an observation of a single object. But during these observations, you might have different observing conditions and decide to split them, with independent IRFs. Independent to event types, we need to find way (???) what to dow with OBS_ID.
+Question came up already, as I have heard, during discussion of VODF. What does the OBS_ID signify? Bruno and Karl redefine the OBS_ID. 
+For CTAO, undefined. We have to take care about event times and stable obs times.
+
+RT: In our internal model we need probably one unique identifier, maybe it can match OBS_ID, maybe not. This means, suggested to the presentation of LS, that converter has to do this. Not clear, if two different observations for different ops-types (???). Design choice we have to make. Internal obstable will depend on these kind of choices. In general, my suggestion: We have some observation metadata and ideally the observation table should follow those as much as possible. Currently only obs_id is mandatory entry in the metadata. In the end, maybe it should be the same for the table. But you have a Pointing, RADEC, AltAz, ... This could basically serve as a kind of building block for the datamodel.
```

Comment:
```suggestion
RT: In our internal model we need probably one unique identifier, maybe it can match OBS_ID, maybe not. This means, suggested to the presentation of LS, that converter has to do this. Not clear, if two different observations for different event types. Design choice we have to make. Internal obstable will depend on these kind of choices. In general, my suggestion: We have some observation metadata and ideally the observation table should follow those as much as possible. Currently only obs_id is mandatory entry in the metadata. In the end, maybe it should be the same for the table. But you have a Pointing, RADEC, AltAz, ... This could basically serve as a kind of building block for the datamodel.
```